runhouse/main.py

```diff
@@ -155,46 +154,21 @@ def cluster_ssh(cluster_name: str):
         c.ssh()
 
     except ValueError:
-        import sky
+        try:
+            import sky
 
-        from runhouse import OnDemandCluster
-        from runhouse.constants import DEFAULT_SSH_PORT
-        from runhouse.resources.hardware.utils import _run_ssh_command
-
-        state = sky.status(cluster_names=[cluster_name], refresh=False)
+            state = sky.status(cluster_names=[cluster_name], refresh=False)
```

Comment:
i guess it was already like this before, but running sky status here is really only relevant if the cluster's launcher is set to local no?

Reply:
oh i see, this is if we fail to even load it. ya i guess may as well try checking with local sky

Reply:
yea, if the launcher is den the cluster should be saved/loaded down and run through the first part? but can also make it explicit here

Reply:
i guess most explicit would be to exit if a value error is raised and launcher is set to den

Reply:
actually if we can't load a cluster, we can't determine for sure if it was supposed to be launched with den or local, so I think leaving it probably handles the cases already (theoretically will be saved/loaded. and if for some reason it isn't, it won't be found locally if launched through den, and will raise an error then)